services/streammanager/lambda/lambdamanager.go

```diff
@@ -71,9 +71,14 @@ func (producer *LambdaProducer) Produce(jsonData json.RawMessage, _ interface{})
 	if input.Payload == "" {
 		return 400, "Failure", "[Lambda] error :: Invalid payload"
 	}
-	config := input.DestinationConfig
-	if config == nil {
-		return 400, "Failure", "[Lambda] error :: Invalid destination config"
+	var config destinationConfig
+	err = mapstructure.Decode(destConfig, &config)
+	if err != nil {
+		returnMessage := "[Lambda] error while unmarshalling destConfig :: " + err.Error()
+		return 400, "Failure", returnMessage
+	}
+	if config.InvocationType == "" {
+		config.InvocationType = "Event"
```

Comment:
Is this change relevant? Also, is there a test case for this?

Reply:
Yes this required as this not part of the destConfig, we are hard coding it.

Reply:
We are always sending this from transformer there won't be any scenario of this being empty string

Reply:
@utsabc the plan is to not use the config sent from the transformer.

Reply:
We won't be sending this in the future from transformer so this required here.

Reply:
Got it